src/frontend/src/header/login/index.jsx

```diff
@@ -19,12 +19,14 @@ export default function Popup({ changePopup, onAuthorization }) {
   const [send, onSend] = useState(false);
   const [remember, rememberMe] = useState(false);
   const [show, onShow] = useState(false);
```

Comment:
please, try to change styles of writing useState
for a example - [send, setSend] = useState(false). Do not use [send, onSend].
look through all places and change in according style. 
You can look in docs how they use hooks - https://legacy.reactjs.org/docs/hooks-state.html